CHANGES.rst

```diff
@@ -55,6 +55,10 @@ astropy.io.votable
 astropy.modeling
 ^^^^^^^^^^^^^^^^
 
+- Major rework of modeling internals.
+  See modeling documentation for details.
+  `<https://docs.astropy.org/en/v4.0/modeling/changes_for_4.html>`_ . [#8769]
```

Comment:
this needs to be changed to an intersphinx link to the relevant what's new section, etc, but that can be done when that docs is written up before the release.